learnersync.go

```diff
@@ -169,11 +169,37 @@ func (s *Sync) Close() {
 
 func (s *Sync) ignorable(path string) bool {
 	for _, pattern := range s.Ignore {
+		// Strip trailing slash from pattern to handle "node_modules/" like "node_modules"
+		pattern = strings.TrimSuffix(pattern, "/")
+
+		// First, try matching against the full path
 		matched, err := filepath.Match(pattern, path)
 		fatalIfSet(err)
 		if matched {
 			return true
 		}
+
+		// If pattern doesn't contain wildcards, also check if it matches any path component
+		// This allows "node_modules" to match "/app/exercises/node_modules" and files within it
```

Comment:
Would this case not be caught by the `Match` above? Or is `Match` from the start of the string.